stories/src/multi-drag/reorder.old.js

```diff
@@ -0,0 +1,232 @@
+// @flow
+import type { Column, Entities } from './types';
+import type { Task, Id } from '../types';
+import type { DraggableLocation } from '../../../src/types';
+import reorder from '../reorder';
+
+type Args = {|
+  entities: Entities,
+  selectedTaskIds: Id[],
+  source: DraggableLocation,
+  destination: DraggableLocation,
+|}
+
+export type Result = {|
+  entities: Entities,
+  // a drop operations can change the order of the selected task array
+  selectedTaskIds: Id[],
+|}
+
+type Entry = {|
+  task: Task,
+  columnId: string,
+  index: number,
+  selectionIndex: number,
+|}
+
+const getColumnById = (columns: Column[], id: string): Column => {
+  const column: ?Column = columns.find((col: Column) => col.id === id);
+  if (!column) {
+    throw new Error('cannot find column');
+  }
+  return column;
+};
+
+const withNewTasks = (column: Column, tasks: Task[]): Column => ({
+  id: column.id,
+  title: column.title,
+  tasks,
+});
+
+const reorderMultiDrag = ({
+  columns,
+  selected,
+  source,
+  destination,
+}): Result => {
+  // 1. remove all of the selected tasks from their lists
```

Comment:
This is the only numbered comment in the function. Should we remove the number or add some to the other comments?